### PR TITLE
Fix problem: pfx file not committed.

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -228,6 +228,7 @@ ClientBin/
 *.dbproj.schemaview
 *.jfm
 *.pfx
+!/aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/resources/encrypted-private-key-and-certificate.pfx
```
Comment (backwind1233) [inline .gitignore:231]: I see

**Review by backwind1233 — APPROVED** [review]
LGTM

File: .gitignore [hunk 2]
```diff
@@ -228,6 +228,7 @@ ClientBin/
 *.dbproj.schemaview
 *.jfm
 *.pfx
```
Comment (backwind1233) [inline .gitignore:230]: can we remove this line instead of adding a new line?

**Review by backwind1233 — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (chenrujun) [reply .gitignore:230]: No. 
`pfx` the the certificate file type. By default, certificate should not be committed., just like secrets should not be committed.

**Pull request merged**